yt/frontends/athena_pp/fields.py

```diff
@@ -81,11 +81,7 @@ def _thermal_energy(field, data):
                            units=unit_system["specific_energy"])
         # Add temperature field
         def _temperature(field, data):
-            if data.has_field_parameter("mu"):
-                mu = data.get_field_parameter("mu")
-            else:
-                mu = 0.6
-            return (data["gas","pressure"]/data["gas","density"])*mu*mh/kboltz
+            return (data["gas","pressure"]/data["gas","density"])*data.ds.mu*mh/kboltz
```

Comment:
Looks like above you're setting `self.parameters["mu"]`. Do you need to use that here?

Reply:
Ah, I didn't do this right. Thanks for the find again.